cmd/fyne/internal/mobile/dex.go

```diff
@@ -6,179 +6,183 @@
-	`xlT3BlbgAOZG9TaG93S2V5Ym9hcmQAAWUAIWV4Y2VwdGlvbiByZWFkaW5nIEtleUNoYXJh` +
-	`Y3Rlck1hcAASZmlsZVBpY2tlclJldHVybmVkAAxmaW5kVmlld0J5SWQAA2dldAAPZ2V0QW` +
-	`Jzb2x1dGVQYXRoAA9nZXRBY3Rpdml0eUluZm8AC2dldENhY2hlRGlyAAxnZXRDb21wb25l` +
-	`bnQAB2dldERhdGEADGdldERlY29yVmlldwAJZ2V0SGVpZ2h0AAlnZXRJbnRlbnQAEWdldF` +
-	`BhY2thZ2VNYW5hZ2VyAAtnZXRSb290VmlldwATZ2V0Um9vdFdpbmRvd0luc2V0cwAHZ2V0` +
-	`UnVuZQAJZ2V0U3RyaW5nABBnZXRTeXN0ZW1TZXJ2aWNlABpnZXRTeXN0ZW1XaW5kb3dJbn` +
-	`NldEJvdHRvbQAYZ2V0U3lzdGVtV2luZG93SW5zZXRMZWZ0ABlnZXRTeXN0ZW1XaW5kb3dJ` +
-	`bnNldFJpZ2h0ABdnZXRTeXN0ZW1XaW5kb3dJbnNldFRvcAAJZ2V0VG1wZGlyAAhnZXRXaW` +
-	`R0aAAJZ2V0V2luZG93AA5nZXRXaW5kb3dUb2tlbgAcZ2V0V2luZG93VmlzaWJsZURpc3Bs` +
-	`YXlGcmFtZQAQZ29OYXRpdmVBY3Rpdml0eQAGaGVpZ2h0AAxoaWRlS2V5Ym9hcmQAF2hpZG` +
-	`VTb2Z0SW5wdXRGcm9tV2luZG93AAxpbnB1dF9tZXRob2QADWluc2V0c0NoYW5nZWQADmtl` +
-	`eWJvYXJkRGVsZXRlAA1rZXlib2FyZFR5cGVkAARsZWZ0AAZsZW5ndGgABGxvYWQAC2xvYW` +
-	`RMaWJyYXJ5ABJsb2FkTGlicmFyeSBmYWlsZWQAJ2xvYWRMaWJyYXJ5OiBubyBtYW5pZmVz` +
-	`dCBtZXRhZGF0YSBmb3VuZAAJbVRleHRFZGl0AAhtZXRhRGF0YQAEbmFtZQAIb2xkU3RhdG` +
-	`UAEG9uQWN0aXZpdHlSZXN1bHQACG9uQ3JlYXRlAA5vbkxheW91dENoYW5nZQANb25UZXh0` +
-	`Q2hhbmdlZAAIcHV0RXh0cmEADHJlcXVlc3RGb2N1cwADcnVuAA1ydW5PblVpVGhyZWFkAA` +
-	`1zZXRJbWVPcHRpb25zAAxzZXRJbnB1dFR5cGUAD3NldExheW91dFBhcmFtcwAHc2V0VGV4` +
-	`dAAHc2V0VHlwZQANc2V0VmlzaWJpbGl0eQAKc2V0dXBFbnRyeQAMc2hvd0ZpbGVPcGVuAA` +
-	`xzaG93S2V5Ym9hcmQADXNob3dTb2Z0SW5wdXQABXNwbGl0ABZzdGFydEFjdGl2aXR5Rm9y` +
-	`UmVzdWx0AAtzdWJTZXF1ZW5jZQAGdGhpcyQwAAZ0aGlzJDEACHRvU3RyaW5nAAN0b3AAIn` +
-	`Vua25vd24ga2V5Ym9hcmQgdHlwZSwgdXNlIGRlZmF1bHQADHVwZGF0ZUxheW91dAAQdmFs` +
-	`JGtleWJvYXJkVHlwZQAFdmFsdWUABXdpZHRoAAF8AE4CAAAHDgBRAAd3EAIOWXmWl3i0lp` +
-	`aXpZYCbSwgSwJ0HQB7AQAHDgB+AAcOtADAAQEABw4AwwEJAAAAAAAAAAAABw5aANUBAQAH` +
-	`DgDnAQEABw4A4wEEAAAAAAcOANgBBAAAAAAHDgESDwEfE7QCex0AyQEBAAcOAMwBAAcd4b` +
-	`TEW5a1AhTgAC4ABw44Ti0AGgEABw4AGgIAAAcOABoBAAcOABoCAAAHDgAaAAcOABoCAAAH` +
-	`DgCVAQMAAAAHHYc0AnssIB6DAHMABw5aAKwBAAcOS6NMS38Cex2HSx4AyQEABw4CIoYAhA` +
-	`EBAAcOWgBKAQAHDloAdwAHDoe0iIwAiAEBAAcOeOFat1qlGQBOAQAHDgIihgAzAAcOAPAB` +
-	`AwAAAAcOAgw7AnkdPGxLALsBAQAHDjw8PbSMADgABw7DAg4sAnYdh4VMHlq1tP/QAAIZAb` +
-	`ABGkoCGgJVBACSAR4CGQGwARpIAhkBsAEaWgIZAbABGj8CGQGwARpcBUSQAAgEAAQBBAIE` +
-	`AQACAQEEkCABkCA0gIAEsBY1AdAWAAEBAQaQIDaAgATQGDcB7BgAAQEBB5AgOICABJQZOQ` +
-	`GwGQABAQMIkCA6gIAEzBk7AegZAQH8GQEBkBoAAQEBCZAgPoCABMAbPwHcGwYCEQcKGgEa` +
-	`ARoBGgEaAQoQAgECQIGABIgdAYggrB0BiCDEHQGIINwdAYgg9B0BiCCMHgGIIKQeBYICAA` +
-	`UIvB4ECJAfAYICAAGCAgABggIAAQKsHwQCsCABCNQgAQjwIEgAjCEBAOQhAQDoIggAjCMH` +
-	`BLAjAQH0IwYAwCQRAAAAAAAAAAEAAAAAAAAAAQAAALMAAABwAAAAAgAAAC0AAAA8AwAAAw` +
-	`AAADkAAADwAwAABAAAABIAAACcBgAABQAAAGEAAAAsBwAABgAAAAYAAAA0CgAAAxAAAAUA` +
-	`AAD0CgAAASAAACAAAAAwCwAABiAAAAUAAAAcEwAAARAAAB8AAABsEwAAAiAAALMAAACIFA` +
-	`AAAyAAACAAAACGIAAABCAAAAYAAADTIQAABSAAAAEAAAD/IQAAACAAAAYAAAAMIgAAABAA` +
-	`AAEAAADoIgAA` +
+var dexStr = `ZGV4CjAzNQAClnuWucRCmShMpICNQdPrq7aZEIeogCKMJAAAcAAAAHhWNBIAAAAAAAAAAL` +
```

Comment:
What is this doing and why is it changed? Unfortunately just seems like a bunch of scramble characters to an untrained eye :sweat_smile: A comment might be a good addition :)

Reply:
The comment is at the top of the file, it is generated by gendex.
Every time we change the Java bootstrap code in the mobile dep we have to re-package the compiled object code into this "dex" file (android talk for java bytecode).

Reply:
Sorry. I thought I was looking at the top of the file when I saw the `package main`.